astropy/units/astrophys.py

```diff
@@ -169,6 +169,14 @@
          doc="Reduced/\"dimensionless\" Hubble constant",
          format={'latex': r'h_{100}'})
 
+# The torr is almost the same as mmHg but not quite.
+# See https://en.wikipedia.org/wiki/Torr
+# Define the unit here despite it not being an astrophysical unit.
+# It may be moved if more similar units are created later.
+def_unit(['Torr', 'torr'], _si.atm.value/760. * si.Pa, namespace=_ns,
+         prefixes=True,
```

Comment:
I would do only `milli`, since there is no need to pollute the namespace with units that are never used.

Reply:
I'm asking the vacuum engineers whether they are okay with Torr and mTorr.